### DISCUSSION TITLE
Handle Long Table of Content (TOC) in Reveal JS

### DISCUSSION THREAD

**Discussion by RoyiAvital** [quarto-dev/quarto-cli#3743] [Q&A]
I have a long presentation with many slides.
Hence the Table of Content (ToC) overflows the page (I set `toc-depth: 2` so 90% and use `.unlisted` heavily of the slides doesn't appear, yet still).

I was thinking about 3 possible solutions:
1. Make it scrollable.
2. Change the font size.
3. Separate into 2 columns.

I was wondering if there is a way to have any of the options above.
I looked on the [RevelJS Reference](https://quarto.org/docs/reference/formats/presentations/revealjs.html) and couldn't find any way beyond setting the depth.

**Comment by skwde** [comment]
Did you find a way to solve this?

Another useful option might be to allow for two column tocs. Though this only works for short headings.

**Reply by skwde** [reply]
In the meantime I found this related discussion handling the font size aspect.

https://github.com/quarto-dev/quarto-cli/discussions/6087
